src/test/regress/expected/citus_split_shard_columnar_partitioned.out

```diff
@@ -0,0 +1,507 @@
+ sensorscolumnar_news_8970007                   | CREATE UNIQUE INDEX sensorscolumnar_news_pkey_8970007 ON citus_split_test_schema_columnar_partitioned.sensorscolumnar_news_8970007 USING btree (measureid, eventdatetime, measure_data)
+ sensorscolumnar_old_8970005                    | CREATE INDEX sensorscolumnar_old_lower_idx_8970005 ON citus_split_test_schema_columnar_partitioned.sensorscolumnar_old_8970005 USING btree (lower((measureid)::text))
+ sensorscolumnar_old_8970005                    | CREATE UNIQUE INDEX sensorscolumnar_old_pkey_8970005 ON citus_split_test_schema_columnar_partitioned.sensorscolumnar_old_8970005 USING btree (measureid, eventdatetime, measure_data)
+(22 rows)
+
+    SELECT stxname FROM pg_statistic_ext
+    WHERE stxnamespace IN (
+        SELECT oid
+        FROM pg_namespace
+        WHERE nspname IN ('citus_split_test_schema_columnar_partitioned')
+    )
+    ORDER BY stxname ASC;
+   stxname
+---------------------------------------------------------------------
+ s1
+ s1_8970000
+ s1_c
+ s1_c_8970004
+ s2
+ s2_8970002
+ s2_c
+ s2_c_8970006
+(8 rows)
+
+    \c - - - :worker_2_port
+    SET search_path TO "citus_split_test_schema_columnar_partitioned";
+    SET citus.show_shards_for_app_name_prefixes = '*';
+    SELECT tbl.relname, fk."Constraint", fk."Definition"
+            FROM pg_catalog.pg_class tbl
+            JOIN public.table_fkeys fk on tbl.oid = fk.relid
+            WHERE tbl.relname like '%_89%'
+            ORDER BY 1, 2;
+ relname | Constraint | Definition
+---------------------------------------------------------------------
+(0 rows)
+
+    SELECT tablename, indexdef FROM pg_indexes WHERE tablename like '%_89%' ORDER BY 1,2;
+        tablename        |                                                                     indexdef
+---------------------------------------------------------------------
+ reference_table_8970011 | CREATE UNIQUE INDEX reference_table_pkey_8970011 ON citus_split_test_schema_columnar_partitioned.reference_table_8970011 USING btree (measureid)
+(1 row)
+
+    SELECT stxname FROM pg_statistic_ext
+    WHERE stxnamespace IN (
+        SELECT oid
+        FROM pg_namespace
+        WHERE nspname IN ('citus_split_test_schema_columnar_partitioned')
+    )
+    ORDER BY stxname ASC;
+ stxname
+---------------------------------------------------------------------
+ s1
+ s1_c
+ s2
+ s2_c
+(4 rows)
+
+-- END: Show the current state on workers
+-- BEGIN: Split a shard along its co-located shards
+\c - - - :master_port
+    SET search_path TO "citus_split_test_schema_columnar_partitioned";
+    SET citus.next_shard_id TO 8999000;
+    SELECT nodeid AS worker_1_node FROM pg_dist_node WHERE nodeport=:worker_1_port \gset
+    SELECT nodeid AS worker_2_node FROM pg_dist_node WHERE nodeport=:worker_2_port \gset
+    SELECT pg_catalog.citus_split_shard_by_split_points(
+        8970000,
```

Comment:
wondering whether we should try to split both parent and child (should have same effect)

Reply:
Done.